src/App.css

```diff
@@ -1,21 +1,285 @@
+  width: 0.125rem;
+  height: 1.5rem;
+  margin-right: 15px;
+  border: solid 1px rgba(218, 215, 215, 0.473);
+}
+
+.oval-2 {
+  width: 4.25rem;
+  height: 4.25rem;
+}
+
+.percentage {
+  align-items: center;
+  gap: 10px;
+}
+
+.pernum {
+  font-family: 'Nunito', sans-serif;
+  font-size: 2rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.pertext {
+  font-family: 'Nunito', sans-serif;
+  font-size: 0.875rem;
+  font-weight: normal;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+}
+
+.big-sep {
+  width: 0.125rem;
+  height: 5.5rem;
+  margin-right: 15px;
+  border: solid 1px rgba(218, 215, 215, 0.473);
+}
+
+.book-others {
+  justify-content: space-between;
+  gap: 40px;
+}
+
+.current-chapter {
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 5px;
+}
+
+.chap-text {
+  font-family: 'Nunito', sans-serif;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #9e9d9d;
+}
+
+.chap-sub-text {
+  font-family: 'Nunito', sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+}
+
+.chap-sub-num {
+  font-family: 'Nunito', sans-serif;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+}
+
+.chap-num {
+  gap: 5px;
+}
+
+.chap-btn {
+  width: 11.5rem;
+  height: 2.063rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  color: #fff;
+  border: none;
+  outline: none;
+  cursor: pointer;
+}
+
+.add-wrapper {
+  margin: 2.313rem 6.25rem 2.313rem;
+  padding: 2rem 9.188rem 1.625rem 1.688rem;
+  border-radius: 4px;
+  background-color: #fafafa;
   gap: 10px;
 }
+
+.add-title {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+  text-transform: uppercase;
+}
+
+input {
+  height: 2.813rem;
+  margin: 1.188rem 2.125rem 0 0;
+  border-radius: 4px;
+  border: solid 1px #c2c2c24f;
+  background-color: #fff;
+  padding-left: 1rem;
+}
+
+.input-title {
+  width: 45%;
+}
+
+.input-category {
+  width: 30%;
+}
+
+.input-category::placeholder {
+  color: #888;
+}
+
+.input-button {
+  width: 15%;
+  height: 2.813rem;
+  margin: 1.188rem 0 0 2.125rem;
+  padding: 0.801rem 1.188rem 0.886rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
+  outline: none;
+  font-family: 'Nunito', sans-serif;
+  font-size: 0.813rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: 0.5px;
+  text-align: center;
+  text-transform: uppercase;
+  color: #fff;
+}
```

Comment:
- [ ] Kindly add a new line after the closing `}` so the line number becomes 286